### DISCUSSION TITLE
Audio always set to 48 kHz [Mac] / Audio Configuration

### DISCUSSION THREAD

**Discussion by verstaerkermusic** [ggarra13/mrv2#221] [Ideas]
when i start mrv2  it always sets the audio playbackrate to 48 kHz - wich can be a bit annoying as i work with Audiosoftware often simultaneously 

so i'd like to be able to configure Audio 

1.  to use/not use audio at all (or only use it when i actually playback videos with audio)
2.  if a certain sample-playbackrate should be set or not (most apps that playback video doesnt require a specific samplerate  and therefore not request a change)
3. wich soundcard  to use and ideally wich channels or just use whats setup in macOs

**Comment by ggarra13** [comment]
@verstaerkermusic I did not forget about your request, but only had until now to investigate it.

> 1. to use/not use audio at all (or only use it when i actually playback videos with audio)

This can be done in any version of mrv2 if you click on the "Audio/Mute" icon in the timeline; you exit mrv2 and open it again (it should open with the Audio/Mute icon set to Mute).  Report if this does not work for you.  I should probably move the setting to the new Window->Preference->Audio section,

> 2. if a certain sample-playbackrate should be set or not (most apps that playback video doesnt require a specific samplerate  and therefore not request a change)

AFAIK, this cannot currently be done, as we need to set the sample rate to the highest value to play .otio movies, which may mix sample rates during playback.

> 3. wich soundcard  to use and ideally wich channels or just use whats setup in macOs

I've added this functionality in beta of v1.1.8.  You can download the beta from:

https://sourceforge.net/projects/mrv2/files/beta/

To try it, you should go to:

Window->Preferences

and then select:

Audio, Output Device.

I tested it with my earphones and with my MacBook Pro speakers, as I don't have multiple audio cards on my Mac.
Please, can you give it a try and report back if it works for you or there are issues?
